.github/workflows/sync-to-oss.yml

```
@@ -0,0 +1,314 @@
+          token: ${{ secrets.GITHUB_TOKEN }}
+          fetch-depth: 0
+
+      - name: 🔧 Setup Git
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: 📋 Prepare OSS distribution files
+        run: |
+          echo "🎯 Preparing OSS distribution from maria_code to maria..."
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)